boskos/configs.yaml

```diff
@@ -13,6 +13,11 @@ configs:
           version: 1.9
           zone: us-central1-f
           enablekubernetesalpha: true
+        - machinetype: n1-standard-2
+          numnodes: 4
+          version: 1.9
+          zone: us-west1-a
```

Comment:
for multicluster, I think we need the 2 gke projects to be in the same zone.

Reply:
They will be in the same project just in a different region.